services/dkg/pedersen/controller/action.go

```diff
@@ -33,8 +34,14 @@ type initAction struct {
 }
 
 // Execute implements node.ActionTemplate. It creates an actor from
-// the dkgPedersen instance
+// the dkgPedersen instance and links it to an election.
 func (a *initAction) Execute(ctx node.Context) error {
+	electionIDBuf, err := hex.DecodeString(ctx.Flags.String("electionID"))
+	if err != nil {
+		return xerrors.Errorf("failed to decode electionID: %v", err)
+	}
+	// TODO: Check that election ID corresponds to a real election
+
 	var dkgPedersen dkg.DKG
 	err := ctx.Injector.Resolve(&dkgPedersen)
```

Comment:
This fails to compile: `no new variables on left side of :=`. This construct is used throughout the project; any idea what's wrong?

Reply:
This is because the `:=` assumes that `err` hasn't been defined before, which it has. Changing it to `=`